config/application.rb

```diff
@@ -8,14 +8,12 @@
 Bundler.require(*Rails.groups)
 
 # Load dotenv only in development environment
-Dotenv::Railtie.load if ['development', 'test'].include? ENV['RAILS_ENV']
+Dotenv::Rails.load if ['development'].include? ENV['RAILS_ENV']
```

Comment:
This file has many changes but not enough documentation of why it is changed.

Reply:
for this specific line, it's because of the Dotenv library upgrade, and when we're upgrading to `3.1.0` last time, this `Dotenv::Railtie` should have been changed to `Dotenv::Rails` since the former one is marked as deprecated already. Refer to https://github.com/bkeepers/dotenv/blob/v3.1.2/lib/dotenv/rails.rb#L113

Reply:
Alright, and why the removal of `'test'` here?